server/Orders.js

```diff
@@ -0,0 +1,134 @@
+/* Orders is observer of: Inventory
+ *           observed by: Drivers
+ */
+class Orders {
```

Comment:
rename to ordersResource

resources are typically used to communicate with external services.